src/Dapr.Jobs/Extensions/EndpointRouteBuilderExtensions.cs

```diff
@@ -49,9 +49,8 @@ public static IEndpointRouteBuilder MapDaprScheduledJobHandler(this IEndpointRou
             ReadOnlyMemory<byte> payload = new();
             if (context.Request.ContentLength is > 0)
             {
-                using var memoryStream = new MemoryStream();
-                await context.Request.Body.CopyToAsync(memoryStream, cancellationToken);
-                payload = memoryStream.ToArray();
+                using var streamContent = new StreamContent(context.Request.Body);
+                payload = await streamContent.ReadAsByteArrayAsync(cancellationToken);
```

Comment:
Should work just as well, though I was specifically thinking of (I think) `context.Request.Body.ReadAsByteArrayAsync()`.

Reply:
Haha, whoops. For some reason, Intellisense isn't showing that as an option - I'll have to look into why.